### ISSUE TITLE
Error on training code

### ISSUE THREAD

**Issue by parkjaewoo0611** [liuyuan-pal/Gen6D#13]
Thank you for sharing your great work!
I found an error on the training detector with

python train_model.py --cfg configs/detector/detector_train.yaml

which returns 

Exception has occurred: AttributeError
'GoogleScannedObjectDatabase' object has no attribute 'model_name'

on line 476 of dataset/database.py.

Actually, I found that the self._get_diameter() in line 464 requires self.model_name, which is not declared yet.

Also, how much time does it takes to train each detector, selector, and refiner with your setting?

Any help would be appreciated.

**Comment by liuyuan-pal** [comment]
Thank you for your interest in our work!

The problem is caused by that a line is missed here:
https://github.com/liuyuan-pal/Gen6D/blob/2e39b25e3cd72cf637b97683a28f45b5c29b6890/dataset/database.py#L438
I've fixed this bug so that it is able to compute this data.
These cached data like `object_center.pkl` `object_vert.pkl` should already be included in the downloaded `google_scanned_object_processed.tar.gz`.

About the training time:
On a single 1080ti GPU, the training time for the detector is about 2 days, the selector is about 3 days and the refiner is about 10 hours.

**Comment by parkjaewoo0611** [comment]
Thank you for your reply and update!
Now the detector is being training well without any problems.
I also found an error occur in 

python train_model.py --cfg configs/selector/selector_train.yaml

which returns

Exception has occurred: AssertionError
exception: no description

in 304 line of database.py.
And find out that the database.database_name is 'genmop/tformer-ref' (not starts with 'linemod').
The error might be caused by the validation set based on both linemod/cat and genmop/tformer-ref.

Any help would be appreciated.

**Comment by liuyuan-pal** [comment]
Sorry, this is caused by a small typo, which is also fixed now.

**Comment by parkjaewoo0611** [comment]
Thank you for your fast reply!
Training the selector works perfectly now. 
I also found a small typo in README.md, Training section 7. train the refiner.

python train_model.py --cfg configs/selector/refiner_train.yaml
should be fixed to 
python train_model.py --cfg configs/refiner/refiner_train.yaml

with the fixed command, it works perfectly.
Thank you again for your fast reply and for sharing great codes. 
Close the issue, since the problems were solved.
